File: 프로그래머스/0/120860. 직사각형 넓이 구하기/직사각형 넓이 구하기.js
function solution(dots) {
    const [[x1, y1],[x2, y2],[x3, y3],[x4, y4]] = dots
    
    const x = Math.max(y1, y2, y3, y4) - Math.min(y1, y2, y3, y4)
    const y = Math.max(x1, x2, x3, x4) - Math.min(x1, x2, x3, x4)
    return x * y;
}

File: 프로그래머스/0/120860. 직사각형 넓이 구하기/직사각형 넓이 구하기.py
def solution(dots):
    x_coords = [dot[0] for dot in dots]
    y_coords = [dot[1] for dot in dots]
    
    width = max(x_coords) - min(x_coords)
    height = max(y_coords) - min(y_coords)
    
    return width * height